### ISSUE TITLE
State of the art yolov4 models

### ISSUE THREAD

**Issue by Goru1890** [AlexeyAB/darknet#6988]
I'm writing a state of the art of yolov4 darknet models, so I'm curious to know which is the map, speed and precision from every proposed yolov4 model with COCO 2017.

There are some tables in the wiki page **YOLOv4 model zoo**, but there is no information about how does the subdivision parameter (8, 16, 32 or 64) change the detection precision. 

I also want to make some type of graph comparing map precision (so computational requirements needed) with amazon web service prices because not everyone has a V100 on his workstation. Maybe also one comparing map with graphic card prices. 

Thank you very much for your help.

**Comment by LukeAI** [comment]
check page 7 of this paper https://arxiv.org/pdf/2011.08036.pdf

There's no simple answer re. exactly how subdivisions will affect map. it depends on many things - if you want to make some graphs I think you'll have to choose to fix a bunch of hyperparams and measure yourself.

**Comment by AlexeyAB** [comment]
`mini_batch = batch / subdivisions`
The higher mini_batch - the higher final accuracy - the more GPUs and more expensive GPU you should be using

So should be:
* for YOLOv4 with mosaic=1, should be `mini_batch >= 4`
* for other models should be `mini_batch >= 16`
* to achieve stated results for EfficientDet - you should use `mini_batch=128` with many GPUs/TPUs with total GPU-VRAM size = 512 GB

Read point `Features of training various neural networks` at: https://medium.com/@alexeyab84/yolov4-the-most-accurate-real-time-neural-network-on-ms-coco-dataset-73adfd3602fe

![1_f2diI7O28j2A875FwQPMJA](https://user-images.githubusercontent.com/4096485/99395895-1e458000-28f2-11eb-9dc6-cd303919e6d2.jpeg)
